File: t2/SEP3T2API/SEP3T2API/SEP3T2GraphQL/Models/Message.cs
using System;

namespace SEP3T2GraphQL.Models
{
    public class Message
    {
        public User Sender { get; set; }
        public User Receiver { get; set; }
        public string Content { get; set; }
        public DateTime TimeSent { get; set; } =
            new DateTime(DateTime.Now.Year, DateTime.Now.Month, DateTime.Now.Day, DateTime.Now.Hour,
                DateTime.Now.Minute, DateTime.Now.Second);
    }
}

File: t3/viabnb.tier3.dataserver.rest/src/main/java/dk/viabnb/sep3/group6/dataserver/rest/t3/models/Message.java
package dk.viabnb.sep3.group6.dataserver.rest.t3.models;

import java.time.LocalDateTime;

public class Message
{
  private User sender;
  private User receiver;
  private String content;
  private LocalDateTime timeSent;

  public Message(User sender, User receiver, String content,
      LocalDateTime timeSent)
  {
    this.sender = sender;
    this.receiver = receiver;
    this.content = content;
    this.timeSent = timeSent;
  }

  public User getSender()
  {
    return sender;
  }

  public User getReceiver()
  {
    return receiver;
  }

  public String getContent()
  {
    return content;
  }

  public LocalDateTime getTimeSent()
  {
    return timeSent;
  }

  public void setSender(User sender)
  {
    this.sender = sender;
  }

  public void setReceiver(User receiver)
  {
    this.receiver = receiver;
  }

  public void setContent(String content)
  {
    this.content = content;
  }

  public void setTimeSent(LocalDateTime timeSent)
  {
    this.timeSent = timeSent;
  }
}
